app/models/crime_application.rb

```diff
@@ -97,33 +97,4 @@ def client_details_complete?
   def passporting_benefit_complete?
     valid?(:passporting_benefit)
   end
-
```

Comment:
#applicant_requires_nino_evidence? has been extracted to be a class within the nino rule file.